Add a basic integration test that ensures the python package is installable

We need a basic sanity test that installing the python package doesn't feel like it did in:
* https://github.com/pallets-eco/flask-debugtoolbar/pull/225

```
Run python -c "from flask import Flask; app = Flask(__name__); from flask_debugtoolbar import DebugToolbarExtension; DebugToolbarExtension(app)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/flask_debugtoolbar/__init__.py", line 6, in <module>
    from packaging import version as version_builder
ModuleNotFoundError: No module named 'packaging'
Error: Process completed with exit code 1.
```

Ideally this gets run under all the versions of python we support, so that if some import works under a newer python but not an older python that gets caught as well.

I assume we can use `pip install -e .` to install the package from the local copy, as we want to test the tip of the PR branch, and not what's available on PyPI.

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -21,6 +21,7 @@ jobs:
           - {name: Linux, python: '3.12', os: ubuntu-latest, tox: py312}
           - {name: Windows, python: '3.12', os: windows-latest, tox: py312}
           - {name: Mac, python: '3.12', os: macos-latest, tox: py312}
+          - {name: Minimal, python: '3.12', os: ubuntu-latest, tox: minimal}
           - {name: '3.11', python: '3.11', os: ubuntu-latest, tox: py311}
           - {name: '3.10', python: '3.10', os: ubuntu-latest, tox: py310}
           - {name: '3.9', python: '3.9', os: ubuntu-latest, tox: py39}

--- a/tox.ini
+++ b/tox.ini
@@ -3,6 +3,7 @@ envlist =
     py27
     py3{12,11,10,9,8,7,6}
     stylecheck
+    minimal
 skip_missing_interpreters = True
 
 [testenv]
@@ -12,6 +13,12 @@ deps =
     Pygments
 commands =
     pytest
+
+[testenv:minimal]
+deps =
+    .
+commands =
+    python -c "from flask_debugtoolbar import DebugToolbarExtension"
 
 [testenv:stylecheck]
 deps =

